src/scenarios/shared/sod.py

```diff
@@ -67,7 +68,12 @@ def runtests(self, scenarioname, dirs, artifact=None):
         ]
         sod_args += dirs.split(';')
 
-        RunCommand(sod_args, verbose=True).run()
+        sod_command = RunCommand(sod_args, verbose=True)
+        sod_command.run()
+        
+        zero_size_regex = rf"({(scenarioname or 'Empty Scenario Name')}\s*-\s*Count\s*\|\s*0\.000\s*count)" # Checks if the overall count is zero    
```

Comment:
The parentheses at the start and end of the regex are redundant since the body only contains one alternative